src/vos/vos_iterator.c

```diff
@@ -800,8 +800,13 @@ vos_iterate_key(struct vos_object *obj, daos_handle_t toh, vos_iter_type_t type,
 		const daos_epoch_range_t *epr, bool ignore_inprogress,
 		vos_iter_cb_t cb, void *arg, struct dtx_handle *dth)
 {
+	struct vos_iter_anchors	*anchors = NULL;
 	vos_iter_param_t	 param = {0};
-	struct vos_iter_anchors	 anchors = {0};
```

Comment:
Yeah, this anchor is huge. I see it's still on stack in many places, especially for vos_aggregate() and vos_discard(), they will be called by RDB which is likely running on a ULT with default stack size.

Maybe we'd examine every anchors on stack to see if it needs be removed from stack.

Reply:
Yeah, there are many other places we can/should address.   This patch may be one of many.

I've got another patch https://github.com/daos-stack/daos/pull/6077 that gives us the ability to sort frames by the biggest offenders.